src/DotLiquid/Tags/For.cs

```diff
@@ -23,6 +23,8 @@ namespace DotLiquid.Tags
     ///      &lt;div {% if forloop.first %}class="first"{% endif %}&gt;
     ///        Item {{ forloop.index }}: {{ item.name }}
     ///      &lt;/div&gt;
+    ///    {% else %}

```

Comment:
Pre-existing issue, not a must-fix: this all belongs to the wiki instead.

Reply:
I see. I guess we better stay consistent with Shopify then.
@microalps has recently been updating the documentation for DotLiquid which needs much input. As this big comment from Shopify source code is explaining how to use the filter, it is basically documentation in the code.
I will resolve this conversation as out of scope for the present PR.